src/reader/config.rs

```diff
@@ -74,7 +74,14 @@ pub struct ParserConfig {
     ///
     /// Note that support for this functionality is incomplete; for example, the parser will fail if
     /// the premature end of stream happens inside PCDATA. Therefore, use this option at your own risk.
-    pub ignore_end_of_stream: bool
+    pub ignore_end_of_stream: bool,
+
+    /// Whether or not non-unicode entity references get replaced with the replacement character
+    ///
+    /// When true, any decimal or hexadecimal character reference that cannot be converted from a
+    /// u32 to a char using [std::char::from_u32](https://doc.rust-lang.org/std/char/fn.from_u32.html)
+    /// will be converted into the unicode REPLACEMENT CHARACTER (U+FFFD).
+    pub lax_unknown_entity_references: bool,
```

Comment:
Let's call this `replace_unknown_entity_references`.

Reply:
Sounds good! I've updated the name.